### DISCUSSION TITLE
Sync issue for 36000 notes

### DISCUSSION THREAD

**Discussion by cheapnet** [vzhd1701/evernote-backup#40] [General]
Hello, thank you for developing this. I have 36000 notes in my Evernote account and I'd like to sync them to my macbook, then I am planning to convert the ENEX files to markdown files and import the markdown files into Obsidian. This tool looks very useful, but it stopped after sync-ing 1800+ notes. Is it related to auth being expired?  How can I sync all the 36000 notes?

Thanks. 

-----------------

Current login will expire at 2024-07-07 07:29:58.
Syncing user notebooks...
  [####################################]  5471171/5471143          
Syncing linked notebooks...
36832 note(s) to download...
Requesting access to 3 linked notebook(s)...
Downloading 36832 note(s)...
  [#-----------------------------------]  1872/36832  2d 06:07:25

**Comment by vzhd1701** [comment]
Evernote has rate limits, so downloading a lot of notes in one go may not be possible. Try running it multiple times with an interval.
